docs/reference/schema-viewer-navigation.md

```diff
@@ -0,0 +1,169 @@
+---
+id: schema-viewer-navigation
+title: Schema Viewer Usage
+---
+
+In this section, we will guide you on navigating the [Schema Viewer](https://smartdatalake.ch/json-schema-viewer/), a powerful tool designed to help you understand the structure and relationships of the SDLB API. Additionally, we will delve into the process of mapping the schema logic to the configuration file, allowing you to effectively write SDLB pipelines.
+
+## User interface
+In the Schema Viewer, the user interface is comprised of three main elements, each serving a distinct purpose to enhance your experience and efficiency.
+
+![Image of schema viewer](../images/schema-viewer-ui.png)
+
+1. **Top Bar**: The Schema Viewer's top bar hosts a crucial feature—the schema version selection dropdown. This feature allows you to effortlessly switch between different versions of the displayed API schema. The search bar is another powerful functionality. It simplifies the process of locating specific elements or relationships within your schema. It's a time-saving tool that helps you pinpoint the exact data you're looking for. Finally, for those who need to work with the schema data in a more flexible format, the 'Download' button, also found in the top bar, is really helpful. With a single click, you can obtain the schema in JSON format, making it easy to read it in a Scala or Python script.
```

Comment:
I personally would start with the most important and most used part first, this is Schema Browser in the center. 
Maybe also to add the description bar on the right hand side.

Reply:
Fixed in commit bfafb8519d3fd648f4bf965e8e948230c8dd0e25